README.md

```diff
@@ -1 +1,273 @@
+
+[![License](http://img.shields.io/:license-mit-blue.svg)](https://github.com/bbtsoftware/AzureDevOpsPolicyConfigurator/blob/develop/LICENSE)
+
+## Information
+
+| | Stable | Pre-release |
+|:--:|:--:|:--:|
+|GitHub Release|-|-|
+
+## Build Status
+
+|Develop|Master|
+|:--:|:--:|
+|-|-|
+
+## Documentation
+
+### Generate current structure
+
+ `AzureDevOpsCofigurator.exe generate` command generates JSON files from the current policy
+ structure. The tool iterates through every Team Project, gets every policy set groups them by
+ repository ans saves in the `policies.json` file. In the main directory, every project is saved
+ to the `projects.json` file, and the tool also gets every type available in the current Team
+ Project, it stores them into the `types.json` file in each project directory.
+
+#### Generation example
+
+```cmd
+AzureDevOpsCofigurator.exe generate --auth ntlm --collectionurl https://tfs.yourserver.ch/DefaultCollection
+```
+
+#### Command arguments
+
+| Option                       | Description
+|------------------------------|---------------------------------------------------------------------------------------------------------------------
+| `--collectionUrl` or `-c`    | The Team Collection URL, e.g.: `tfs.yourserver.ch/DefaultCollection`
+| `--auth` or `-a`             | The authentication method, option values: `ntlm`, `oauth`, `basic`, `azureactivedirectory`
+| `--user` or `-u`             | The user for authentication (optional for PAT authentication, or for `ntlm` authentication with current user)
+| `--password` or `-p`         | The password, PAT or token for the authentication (optional for `ntlm` authentication with current user)
+| `--destination` or `-d`      | optional, default value is `.\Project`
+
+
+### Simulate and execute changes
+
+ `AzureDevOpsCofigurator.exe whatif` and `AzureDevOpsCofigurator.exe execute` command executes
+ a strategy stored in an input JSON file
+
+#### Simulation example
+
+```cmd
+AzureDevOpsCofigurator.exe whatif -a ntlm -c https://tfs.yourserver.ch/DefaultCollection -i Sample\\testfile.json
+```
+
+#### Execution example
+
+```cmd
+AzureDevOpsCofigurator.exe execute -a ntlm -c https://tfs.yourserver.ch/DefaultCollection -i Sample\\testfile.json
+```
+
+#### Command arguments
+
+| Option                       | Description
+|------------------------------|---------------------------------------------------------------------------------------------------------------------
+| `--collectionUrl` or `-c`    | The Team Collection URL, e.g.: `tfs.yourserver.ch/DefaultCollection`
+| `--auth` or `-a`             | The authentication method, option values: `ntlm`, `oauth`, `basic`, `azureactivedirectory`
+| `--user` or `-u`             | The user for authentication (optional for PAT authentication, or for `ntlm` authentication with current user)
+| `--password` or `-p`         | The password, PAT or token for the authentication (optional for `ntlm` authentication with current user)
```

Comment:
What's the difference between PAT or token?

Reply:
Same, OAuth names it simply token, removed